R/deploy-site.R

```diff
@@ -134,7 +155,7 @@ github_worktree_add <- function(dir, branch) {
   rule("Adding worktree", line = 1)
   git("worktree",
     "add",
-    "--track", "-b", branch,
+    "--track", "-B", branch,
```

Comment:
I had to change this to a `-B` because if we have created a new branch above then the worktree add would fail, as the branch already existed locally.